src/tools/simeng/main.cc

```diff
@@ -84,9 +84,10 @@ int main(int argc, char** argv) {
 
   // Get the memory size from the YAML config file.
   const size_t memorySize =
-      Config::get()["Simulation-Memory"]["Size"].as<size_t>();
-  const uint64_t latency =
-      Config::get()["LSQ-L1-Interface"]["Access-Latency"].as<uint64_t>();
+      Config::get()["Memory-Hierarchy"]["DRAM"]["Size"].as<size_t>();
+  const uint16_t latency =
```

Comment:
Add a comment stating why L1 cache latency us used for DRAM

Reply:
Done

Reply:
can we not use DRAM here as well, as mentioned we can specify access latency of DRAM as whatever we want through the config?
lets keep it to 5 till we have a good cache hierarchy setup?

Reply:
Done